src/server/controller/metadata/index.ts

```diff
@@ -15,20 +16,24 @@ import { updateTable } from './updateTable'
 import { updateTableSection } from './updateTableSection'
 
 export const MetadataController = {
```

Comment:
I would prefer in this case, to order by entity first 😅and then within each entity, alphabetically. it makes it easier in case of refactor / extraction..